modules/ximgproc/samples/thinning.cpp

```diff
@@ -19,8 +19,8 @@ int main()
 
     /// Apply thinning to get a skeleton
     Mat img_thinning_ZS, img_thinning_GH;
-    //ximgproc::thinning(img_binary, img_thinning_ZS, THINNING_ZHANGSUEN);
-    //ximgproc::thinning(img_binary, img_thinning_GH, THINNING_GUOHALL);
+    ximgproc::thinning(img_binary, img_thinning_ZS, ximgproc::THINNING_ZHANGSUEN);
+    ximgproc::thinning(img_binary, img_thinning_GH, ximgproc::THINNING_GUOHALL);
```

Comment:
This is wrong, the constants are not under te ximgproc namespace for the moment, and I am still awaiting response on why they are not. So this would break the sample.